Sources/MapboxNavigation/SpriteRepository.swift

```diff
@@ -43,62 +51,69 @@ class SpriteRepository {
         updateSprite(completion: completion)
     }
 
-    func updateRepresentation(for representation: VisualInstruction.Component.ImageRepresentation? = nil, completion: @escaping CompletionHandler) {
+    func updateRepresentation(for representation: VisualInstruction.Component.ImageRepresentation? = nil, completion: @escaping (DownloadError?) -> Void) {
```

Comment:
Probably worth typedefing new completion type.

Reply:
Thanks for the suggestion, the `ImageDownloadCompletionHandler` is added to replace it.